service/kvmconfig/v3/version_bundle.go

```diff
@@ -0,0 +1,51 @@
+package v3
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func VersionBundle() versionbundle.Bundle {
```

Comment:
Every versioned resource folder defines its own version bundle now. This should improve the versioning and prevent certain conflicts or human failures during the versioning process, because with this it is way harder to mess with already frozen versions.